jenkins-pipelines/artifacts-centos8-arm.jenkinsfile

```diff
@@ -8,7 +8,8 @@ artifactsPipeline(
     backend: 'aws',
     region_name: 'eu-west-1',
     provision_type: 'spot_low_price',
-    scylla_mgmt_repo: 'http://downloads.scylladb.com.s3.amazonaws.com/manager/rpm/unstable/centos/master/latest/scylla-manager.repo',
+    scylla_mgmt_repo: '',
```

Comment:
why not to use manager version 2.6?

Reply:
Probably because of the comment below in Line 12

Reply:
@ShlomiBalalis 

Something is off here...
The line you replaced is pointing to master repo of manager 

But the job
https://jenkins.scylladb.com/job/enterprise-2021.1/job/artifacts/job/artifacts-centos8-arm-test/17/

is installing 2.3, you are sure this is the right place to disable installing manager ?